Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -1689,6 +1692,17 @@ def prepare_NANO(self, sequence = "nanoAOD"):
                 self._options.customise_commands = self._options.customise_commands + " \n"
             self._options.customise_commands = self._options.customise_commands + "process.unpackedPatTrigger.triggerResults= cms.InputTag( 'TriggerResults::"+self._options.hltProcess+"' )\n"
 
+    def prepare_NANOGEN(self, sequence = "nanoAOD"):
+        ''' Enrich the schedule with NANOGEN '''
+        # TODO: Need to modify this based on the input file type
```

Comment:
Is there a way I can detect the input data tier? I think this would fail if someone wanted to run only the NanoGen sequence from an input GEN file, for example